### DISCUSSION TITLE
Docs: how to opt back into SWC

### DISCUSSION THREAD

**Discussion by JoshuaKGoldberg** [vercel/next.js#36605] [Ideas]
### What is the improvement or update you wish to see?

https://nextjs.org/docs/messages/swc-disabled explains why a project opted out of SWC, but doesn't explain how to explicitly opt in. Is that possible? (if not, I suppose this is a feature request)

### Is there any context that might help us understand?

Other tools such as Jest and Storybook react to the presense of a `.babelrc`. It's inconvenient -or sometimes not supported well- to have to configure those tools to point to a custom `.babelrc` location.

### Does the docs page already exist? Please link to it.

https://nextjs.org/docs/messages/swc-disabled

**Comment by timneutkens** [comment]
Explicitly opting in is not supported currently so this is indeed a feature request. For Jest you can use: https://nextjs.org/docs/testing#setting-up-jest-with-the-rust-compiler.
